index.html

```diff
@@ -1258,10 +1258,10 @@
         <dd>
           When the payment request is updated using <a data-lt=
           "PaymentRequestUpdateEvent.updateWith">updateWith()</a>, the
-          <a>PaymentDetailsUpdate</a> can contain a message in the <a>error</a>
-          member that will be displayed to the user. For example, this might
-          commonly be used to explain why goods cannot be shipped to the chosen
-          shipping address.
+          <a>PaymentDetailsUpdate</a> can contain a human-readable message in
```

Comment:
("human-readable" was already in use in the spec, which is why I reused it - maybe we can come up with something nicer)